mesh/nurbs.hpp

```diff
@@ -403,6 +405,21 @@ class NURBSExtension
    void UniformRefinement();
    void KnotInsert(Array<KnotVector *> &kv);
    void KnotInsert(Array<Vector *> &kv);
+
+   int GetElementPatch(int elem) const { return el_to_patch[elem]; }
+   void GetElementIJK(int elem, Array<int> & ijk);
+
+   const Array3D<int>& GetPatchDofs(const int patch);
+
+   int NumPatchDofs1D(const int patch, const int dim);
```

Comment:
Should document these methods

Reply:
I added documentation in every place you pointed out.